CMakeLists.txt

```diff
@@ -17,18 +17,28 @@
 # See: https://cmake.org/cmake/help/latest/manual/cmake-policies.7.html
 cmake_minimum_required(VERSION 3.16...3.28)
 
-project("Bitcoin Core"
-  VERSION 26.99.0
+set(PACKAGE_NAME "Bitcoin Core")
+set(CLIENT_VERSION_MAJOR 26)
+set(CLIENT_VERSION_MINOR 99)
+set(CLIENT_VERSION_BUILD 0)
+set(CLIENT_VERSION_RC 0)
+set(CLIENT_VERSION_IS_RELEASE "false")
+set(COPYRIGHT_YEAR "2024")
```

Comment:
Is `COPYRIGHT_YEAR` referring to the year in which the `CMAKE` project will be (most likely) released?
Otherwise could be get dynamically:
```
string(TIMESTAMP CURRENT_YEAR "%Y")
set(COPYRIGHT_YEAR ${CURRENT_YEAR})
```